astacus/coordinator/coordinator.py

```diff
@@ -221,31 +246,82 @@ def get_subresult_url(request_url: URL, op_id: int) -> str:
     return urlunsplit(parts)
 
 
-class Coordinator(op.OpMixin):
-    """ Convenience dependency which contains sub-dependencies most API endpoints need """
-    state: CoordinatorState
+class BackupRestoreOp(CoordinatorOpWithClusterLock):
```

Comment:
This name is pretty bad - it can be also construed as backup restoration (although it is really utility superclass). Could you figure something else for it? As it seems to mostly deal with the steps, perhaps StepOp(Base) or something.